README.md

```diff
@@ -451,8 +450,7 @@ Name                | Description                             | Default
 --------------------|-----------------------------------------|---------------
 [reconnect]         | Configures the exponential back off     | `{}`
 timeout             | Connect time out                        | `10000` ms
-ping                | Ping interval to test connection        | `25000` ms
-pong                | Time the server has to respond to ping  | `10000` ms
+ping                | Max time to wait for a server ping      | `45000` ms
```

Comment:
Default is server generated and depends on `timeout` value but I agree, an old custom value can cause problems.
Suggestions for the names?